tests/components/gree/test_bridge.py

```diff
@@ -69,3 +73,33 @@ async def test_discovery_after_setup(
     device_infos = [x.device.device_info for x in hass.data[GREE][COORDINATORS]]
     assert device_infos[0].ip == "1.1.1.2"
     assert device_infos[1].ip == "2.2.2.1"
+
+
+async def test_coordinator_updates(
+    hass: HomeAssistant, freezer: FrozenDateTimeFactory, discovery, device
+) -> None:
+    """Test gree devices update their state."""
+    await hass.config.async_set_time_zone("UTC")
+    freezer.move_to(dt_util.utcnow())
```

Comment:
Why do we do this? To use the freezer we don't have to freeze time fyi

Reply:
Not needed at all, thanks.